packages/client/docusaurus/docs/javascript/02-guides/07-query-call-members.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: querying-call-members
+title: Querying Call Members
+description: How to query call members
+---
+
+import FilterConditions from '../../../shared/_filter-operators.mdx';
+import CallMemberFilters from '../../../shared/video/_call-member-filters.mdx';
+import CallMemberSort from '../../../shared/video/_call-member-sort-fields.mdx';
```

Comment:
Should these be wrapped in ```typescript ...``` wrapper?

Reply:
No, they work like this, I checked by locally running the docs site

Reply:
Nice, I was checking this with the `preview file` option here.